contractcourt/channel_arbitrator.go

```diff
@@ -1113,17 +1118,17 @@ func (c ChainActionMap) Merge(actions ChainActionMap) {
 // we should go on chain to claim.  We do this rather than waiting up until the
 // last minute as we want to ensure that when we *need* (HTLC is timed out) to
 // sweep, the commitment is already confirmed.
-func (c *ChannelArbitrator) shouldGoOnChain(htlcExpiry, broadcastDelta,
+func (c *ChannelArbitrator) shouldGoOnChain(htlc channeldb.HTLC, broadcastDelta,
```

Comment:
For ease of reviewing, the non-functional refactorings (introducing `Clock` and the function signature change of `shouldGoOnChain`) could be isolated in a separate commit.

Reply:
@joostjager Let me verify I understood it correct:
On top of the original code of this file:
1. First commit for the signature change only.
2. Second commit for the shouldGoOnChain body and introducing Clock (As `Clock` is used in the new implementation).

Was that the suggestion?

Reply:
Ah yes, the `Clock` wasn't already there. In that case it is indeed only the signature change in a separate commit. But up to you whether you think it is worth it, it is non-blocking.